src/main/java/de/unijena/cheminf/mortar/model/fragmentation/algorithm/ErtlFunctionalGroupsFinderFragmenter.java

```diff
@@ -338,7 +340,8 @@ public static enum CycleFinderOption {
      * Constructor, all settings are initialised with their default values as declared in the respective public constants.
      */
     public ErtlFunctionalGroupsFinderFragmenter() {
-        this.settingNameTooltipTextMap = new HashMap(10, 0.9f);
+        int tmpInitialCapacityForSettingNameTooltipTextMap = CollectionUtil.calculateInitialHashCollectionCapacity(6, BasicDefinitions.DEFAULT_HASH_COLLECTION_LOAD_FACTOR);
+        this.settingNameTooltipTextMap = new HashMap(tmpInitialCapacityForSettingNameTooltipTextMap, BasicDefinitions.DEFAULT_HASH_COLLECTION_LOAD_FACTOR);
```

Comment:
Where is this 6 coming from? 
By default a hash map has a size of 16, just to mention it

Reply:
6 is the number of settings this fragmenter has, so the number of elements supposed to be stored in the map. I have introduced a local variable with a name specifying this now.